test/saleB.js

```diff
@@ -81,4 +81,21 @@ contract('Sale', (accounts) => {
       assert.strictEqual(startingBalance.toString(10), finalBalance.toString(10), errMsg);
     });
   });
+
+  it('should allow the owner to withdraw all unsold tokens', async () => {
+    await forceMine(saleConf.endBlock.add(new BN('2', 10)));
+
+    const sale = await Sale.deployed();
+    const startingBalanceSale = await getTokenBalanceOf(sale.address);
+    const owner = await sale.owner.call();
+    const receipt = await sale.withdrawRemainder({ from: owner });
+    const endingBalanceSale = await getTokenBalanceOf(sale.address);
+    const wallet = await sale.wallet.call();
+    const endingBalanceWallet = await getTokenBalanceOf(wallet);
+
+    const errMsg1 = 'Sale does not have 0 tokens, but should.';
+    const errMsg2 = 'Wallet did not receive withdrawn tokens from sale.';
+    assert.equal(endingBalanceSale.toString(10), 0, errMsg1);
```

Comment:
Use strictEqual.

`assert.equal(endingBalanceSale.toString(10), '0', errMsg1);`

(We should do a separate PR to take out all the instances of `assert.equal` in the tests.

Reply:
Fixed